### ISSUE TITLE
make the core logs available on cloud

### ISSUE THREAD

**Issue by h4ck3rk3y** [kurtosis-tech/kurtosis#1513]
### Background & motivation

Its tough to tell why my enclave isn't running or failing as I can't see the logs of core on cloud

### Desired behaviour

I can see the logs of core

### How important is this to you?

Critical; Kurtosis is unusable for me without it.

**Comment by adschwartz** [comment]
@h4ck3rk3y When you say you need to see the "core" logs, are you looking to get the logs from the specific APIC container? If so, this would be a new feature we don't have today as I see it (we only have GetServiceLogs on Engine).

**Comment by h4ck3rk3y** [comment]
Yep this would be getting logs of the API Container! This is an FR
